guides/source/active_record_validations.md

```diff
@@ -236,7 +243,7 @@ irb> p.valid?
 irb> p.errors.objects.first.full_message
 => "Name can't be blank"
 
-irb> p = Person.create
+irb> p = Person.new
```

Comment:
This is a long code block. Maybe a comment above this line to separate the example might help readability? And perhaps naming the `p` to something else, as that variable is declared in line 236 and we are re-declaring it here?